src/hooks/encounter/encounter.ts

```diff
@@ -56,3 +59,28 @@ export const useCreateEncounter = (
 
   return { createEncounter, loading };
 };
+
+const POLLING_INTERVAL = 1000;
+
+export const usePollingForEncounters = (navigation: StackNavigationProp<any>, qrId?: string) => {
+  useEffect(() => {
+    if (!qrId) {
+      return;
+    }
+
+    const intervalId = setInterval(() => {
+      // TODO: use API call here
+      // TODO: check if we need/want a "mounted" variable here
+      const testing = false;
```

Comment:
found out how to test it with setting to `true` here. works fine!

Reply:
:+1: